### ISSUE TITLE
shell command alias and insert output issue

### ISSUE THREAD

**Issue by linc01** [helix-editor/helix#6390]
Hello,

Does the `:insert-output` support user defined shell command alias?

For example, I put `alias myip='curl http://ipecho.net/plain; echo'` in `.zshrc` or create a new file with user defined alias and use it in the `.zshrc` file
```
if [ -f ~/.shell_aliases ]; then
    . ~/.shell_aliases
fi
```
Either way is fine in the terminal, but not work in helix.  when I use `:insert-output myip` , it says `shell error zsh:1:  command not found: myip`. 

How to fix it? Thanks.

**Comment by archseer** [comment]
Yeah the command doesn't run in a shell environment so the aliases never get loaded. You need to invoke it through zsh -c

**Comment by linc01** [comment]
Thanks @archseer, now I use the `.zshenv` to work with helix, it works.
